cocos/math/Vec3.h

```diff
@@ -51,22 +50,22 @@ class CC_DLL Vec3 {
     /**
      * The x-coordinate.
      */
-    float x;
+    float x{};
```

Comment:
float x{0.F} ？

Reply:
member initializer for 'x' is redundant (modernize-use-default-member-init)